controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1661,3 +1661,49 @@ int cli_bml::steering_client_measure(uint32_t steeringGroupIndex, const std::str
     return 0;
 }
 #endif //BEEROCKS_RDKB
+
+int cli_bml::set_dcs_continuous_scan_enable(const std::string &radio_mac, int8_t enable)
+{
+    std::cout << "set_dcs_continuous_scan_enable" << std::endl;
```

Comment:
1. bool instead of uint8_t:
- the BML API uses int - and the CML CLI provides an int - so I don't see a value in int->bool->int conversion
- in the 'get_dcs_scan_results()' I will move the int->bool translation to the caller function and update the signature
2. print the input values:
done

Reply:
OK